server/master/src/main/java/org/apache/accumulo/master/tableOps/ExportTable.java

```diff
@@ -27,12 +31,16 @@
 
   private final ExportInfo tableInfo;
 
-  public ExportTable(String tableName, String tableId, String exportDir) {
+  public ExportTable(String tableName, String tableId, String exportDir) throws ThriftTableOperationException {
     tableInfo = new ExportInfo();
     tableInfo.tableName = tableName;
     tableInfo.exportDir = exportDir;
     tableInfo.tableID = tableId;
-    tableInfo.namespaceID = Tables.getNamespaceId(HdfsZooInstance.getInstance(), tableId);
+    try {
```

Comment:
Could refactor this to resolve the namespace id in FateServiceHandler (using the new method in FateServiceHandler) and just pass the namespaceId to the constructor.